src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/interceptor/UserAuthenticationInterceptor.java

```diff
@@ -35,13 +34,13 @@ public UserAuthenticationInterceptor(LoggingUtils logger) {
      */
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
-        DataMap dataMap = new DataMap.Builder()
-                .requestId(request.getHeader(REQUEST_ID_HEADER_NAME))
-                .status(UNAUTHORIZED.toString())
-                .build();
-
         String identityType = EricHeaderHelper.getIdentityType(request);
         if(identityType == null) {
+            DataMap dataMap = new DataMap.Builder()
```

Comment:
Yes, there's another way to only build the data map when you need it, that does not involve this code duplication. You could have the map built in and returned from a separate function, and call that function in the logging function.